### PR TITLE
chore(web): add cache control max-age

### PR DESCRIPTION
**Pull request by beejjacobs**
## Describe your changes

Ensure static assets have a non-zero max-age applied. To be improved in the future with immutable assets and cache-busting techniques.

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by emil-placheta — APPROVED** [review]

File: packages/forms-web-app/src/config.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ module.exports = {
 			limit: '500kb'
 		}
 	},
+	cacheControl: {
+		maxAge: process.env.CACHE_CONTROL_MAX_AGE || '1d'
```
[the remark this answers is not in the document]
Comment (emil-placheta) [reply packages/forms-web-app/src/config.js:23]: ah, automerged

[the remark this answers is not in the document]
Comment (beejjacobs) [reply packages/forms-web-app/src/config.js:23]: This value doesn't go to the max-age property directly, but `express.static` passes this option to [ms](https://www.npmjs.com/package/ms) package - which can take a string like this

**Pull request merged**